nbs/01_datasets.ipynb

```diff
@@ -252,7 +252,8 @@
     "            if self.shuffle:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #48.                    self.indices = jax.random.permutation(self.key, self.indices)
</pre><p>change to </p><pre class="ql-syntax" spellcheck="false">self.indices = np.random.permutation(self.indices)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/cfnet/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>